src/daniil_efremov_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,21 @@
+def bubble_sort(some_data):
+    dat = list(some_data)
+    len_string = len(dat) - 1
```

Comment:
This variable's name is so misleading. It's said that it's the length of the string, but actually it's not.